src/components/confirm/Confirm.js

```diff
@@ -55,9 +55,10 @@ export class Confirm extends Component {
 			<Dialog open={this.state.active}>
 				<DialogTitle id='alert-dialog-title'>{title}</DialogTitle>
 				<DialogContent>
-					<DialogContentText id='alert-dialog-description'>
```

Comment:
This is commented out because with the translation with components it gives errors as this is a `<p>` tag